### PR TITLE
Travis Cl:  Clean up travis configurations for playwright tests

### LINKED ISSUE #22: Clean up nodejs configuration from  travis file
Clean up nodejs configuration from travis file

## TODO
- [ ] Remove nodejs configuration from travis.yml file
- [ ] Add Github actions badge to the Readme file

### PR DESCRIPTION
**Pull request by hariclerry**
### Tickets
Closes #22

### What does this PR do?
This PR removes nodejs config from .travis.yml file, reverts the file to the previous state, and adds remove travis ci status badge to the README file

### What are the changes made?
- Remove Node configuration from travis.yml file
- Added previous configurations to the file
- Remove travis CI status badge from the Readme file

### Screen shots
None

### CODE REVIEW

**Review by vidhi-mody — CHANGES_REQUESTED** [review]
The .github/workflows/node.js.yml should be in a separate PR. This PR should only contain changes related to the Travis CI cleanup.

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -4,26 +4,31 @@
...
+        steps:
+            - uses: actions/checkout@v2
+            - name: Use Node.js ${{ matrix.node-version }}
+              uses: actions/setup-node@v1
+              with:
+                  node-version: ${{ matrix.node-version }}
+            - name: "Install dependencies"
+              run: |
+                  npm ci
+            - name: "Run eslint"
```
Comment (sovoid) [inline .github/workflows/node.js.yml:29]: Please add this file in a separate PR

**Review by sovoid — CHANGES_REQUESTED** [review]

**Comment by hariclerry** [comment]
Hello @vidhi-mody, those changes are coming in because I rebased with master.

**Comment by sovoid** [comment]
@hariclerry these changes should not be visible here even if they are from master. You must be doing something wrong while rebasing

**Pull request closed without merging**